extensions/kubens/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "kubens",
+  "title": "Switch Kubernetes Namespace",
```

Comment:
Shouldn't the title contain something with `kubens`? I don't know anything about it so it's just a guess 😊

Reply:
Yes. Good point. If you look for a extension you'd search for `kubens`

And you have the description which gives then more context. I'll change itl